### PR TITLE
Qualifying centos 7.8 with dcos 2.0.5 and docker 18.09.9

### PR DESCRIPTION
**Pull request by DominikDary**
Jira: [COPS-6313](https://jira.d2iq.com/browse/COPS-6313)

### CODE REVIEW

**Comment by DominikDary** [comment]
@janisz please review and merge.

**Comment by janisz** [comment]
@DominikDary I'm sorry but I've never committed to this repository and have no idea what's all about.

**Comment by DominikDary** [comment]
@janisz this is now owned by the @dcos/dcos-team , so we need take to care about it.

File: centos/7.8/aws/DCOS-2.0.5/docker-18.09.9-ce/selinux_enforcing/publish_and_test_config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# options: packer_build, dcos_installation, integration_tests, never. Default is dcos_installation. For more details, see README
+# You can also run custom tests with tests_to_run.
+publish_dcos_images_after: integration_tests
+run_framework_tests: false
+run_integration_tests: true
```
Comment (DominikDary) [inline centos/7.8/aws/DCOS-2.0.5/docker-18.09.9-ce/selinux_enforcing/publish_and_test_config.yaml:5]: @wavesoft this is executing the integration tests.

**Pull request merged**